watch.mjs

```diff
@@ -9,17 +9,16 @@ Always use ./build.py to ensure that all projects are built correctly before che
 Also run ./build.py to do any initial repo setup (npm install, copying 3rd party libs, etc.)
 
 Once running, any changes to the source code for Rust directories listed, or for
-the npm, vscode, or playground projects, should automatically recompile. Just
+the npm, vscode, docs, katas, samples or playground projects, should automatically recompile. Just
```

Comment:
To test doc update, I tried modify docs in `library\core\core.qs` and update was reflected in the playground :)